src/components/about/karusellProjektbeteiligte.svelte

```diff
@@ -0,0 +1,253 @@
+<script lang="ts">
```

Comment:
kannst du die ganzen Svelte Dateien erstens auf englisch schreiben und mit einer anderen Benennung (beispielsweise "carousel-projekt-members.svelte" der HTML export würde dann trotzdem CarouselProjektMembers sein) Aber die file halt nicht nach [diesen Best Practices](https://kim-jangwook.medium.com/best-practices-for-organizing-and-structuring-svelte-applications-5f85a3d5a6f5)

Reply:
Bitte nochmal beim nächsten pullrequest überprüfen ob du das so gemeint hattest. Bei dem Befehl konnten keine - als alternativer componentname verwendet werden: "export { default as CarouselProjectParticipants } from './about/carousel-project-participants.svelte';"

Reply:
habe jetzt doch rem verwendet

Reply:
`export { default as CarouselProjectParticipants } from './about/carousel-project-participants.svelte'; `genauso sollte das aussehen. Manchmal liegt das an VS-Code und nicht an dir